src/node/tests/elder.rs

```diff
@@ -706,7 +707,7 @@ impl OtherNode {
 
     fn expect_bootstrap_response(&self) -> BootstrapResponse {
         self.received_messages()
-            .find_map(|msg| match msg.variant {
+            .find_map(|msg| match msg.variant().clone() {
                 Variant::BootstrapResponse(response) => Some(response),
```

Comment:
I'd move the `clone` here, to avoid cloning the variants we don't care about:
```rust
match msg.variant() {
    Variant::BootstrapResponse(response) => Some(response.clone()),
    _ => None,
}
```

Reply:
Good point